src/parser.rs

```diff
@@ -6912,40 +6912,6 @@ mod tests {
         });
     }
 
-    #[test]
-    fn test_parse_limit() {
```

Comment:
I moved this code to sqlparser_common and sqlparser_mysql as appropriate, and used more concise functions to test these dialects